src/main/kotlin/app/revanced/patches/youtube/misc/settings/resource/patch/SettingsResourcePatch.kt

```diff
@@ -77,8 +77,8 @@ class SettingsResourcePatch : AbstractSettingsResourcePatch(
             )
         )
 
-        SettingsPatch.addString("revanced_enable_setting_user_notice_message_title_text",
-            "Do you wish to proceed?")
+        addString("revanced_settings_confirm_user_dialog_title",
+            "Do you wish to proceed?", false)
```

Comment:
Conventionally, strings are added via a host XML file. The `downloads` patch for example uses a [host](https://github.com/revanced/revanced-patches/blob/80c1d45c1cd6bb590d67071a931d908c86ee8a2a/src/main/resources/downloads/host/values/strings.xml) file to [merge](https://github.com/revanced/revanced-patches/blob/80c1d45c1cd6bb590d67071a931d908c86ee8a2a/src/main/kotlin/app/revanced/patches/youtube/interaction/downloads/resource/patch/DownloadsResourcePatch.kt#L55) when patching. This is recommended, so that in the future when new strings are added, we don't have to add multiple `addString` calls.